extensions/amp-ad-network-nws-impl/0.1/test/test-nws-a4a-config.js

```diff
@@ -14,11 +14,19 @@
  * limitations under the License.
  */
 
+// These two are required for reasons internal to AMP
+import '../../../../extensions/amp-ad/0.1/amp-ad-ui';
+import '../../../../extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler';
+
 import {AmpAdNetworkNwsImpl} from '../amp-ad-network-nws-impl';
 import {createElementWithAttributes} from '../../../../src/dom';
+// There seems to be a conflict between import none syntax
```

Comment:
hmm this works for me: 
![image](https://user-images.githubusercontent.com/16087874/99595354-72f80200-29a9-11eb-9e58-9bfb3adcd619.png)

is it possible you have a local rule conflicting?

Reply:
I don't think so, I tried pushing it anyway and the same error appeared in travis: https://travis-ci.com/github/ampproject/amphtml/jobs/443323409

Reply:
ok no worries I can try to fix in a follow up.